lisk-dex-electron/src/App.jsx

```diff
@@ -93,34 +92,34 @@ class App extends React.Component {
       activeMarket: defaultMarketKey,
       activeAssets: configuration.markets[defaultMarketKey].assets,
       enabledAssets: Object.keys(configuration.assets),
-      configurationLoaded: true
+      configurationLoaded: true,
     });
   }
 
-  getTakerOrderIdFromTransaction(transaction) {
+  getTakerOrderIdFromTransaction= (transaction) => {
```

Comment:
There is a missing space after `getTakerOrderIdFromTransaction` - Ideally should be `getTakerOrderIdFromTransaction =`. Is it possible to add it as a rule?